cmd/tink-worker/worker.go

```diff
@@ -155,43 +184,44 @@ func processWorkflowActions(client pb.WorkflowSvcClient) error {
 
 				if err != nil || status != pb.ActionState_ACTION_SUCCESS {
 					if status == pb.ActionState_ACTION_TIMEOUT {
-						log.WithFields(logrus.Fields{"action": action.GetName(), "Task": action.GetTaskName()}).Errorln("Action timed out")
+						ll.With("actionStatus", pb.ActionState_ACTION_TIMEOUT.String())
 						actionStatus.ActionStatus = pb.ActionState_ACTION_TIMEOUT
```

Comment:
I would say to swap these around so it looks like:
```go
actionStatus.ActionStatus = pb.ActionState_ACTION_TIMEOUT
ll.With("actionStatus", actionStatus.ActionStatus.String())						
```
this way if you were to copy the log line somewhere else you won't be logging the wrong thing by mistake.